src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/model/FlowPatch.java

```diff
@@ -48,4 +48,6 @@ public class FlowPatch {
     private String description;
     private FlowEncapsulationType encapsulationType;
     private Set<Integer> vlanStatistics;
+
+    private String correlationId;
```

Comment:
I'm not sure that it should be here. 
FlowPatch is just a model which is used here and in NB. In NB we are setting correclation id to the parent message
```
        FlowPatch flowPatch = flowMapper.toFlowPatch(flowPatchDto);
        flowPatch.setFlowId(flowId);
        CommandMessage request = new CommandMessage(new FlowPatchRequest(flowPatch), System.currentTimeMillis(),
                correlationId);
```
In this case FlowPatch.correlationId will be empty or it will duplicate information from CommandMessage.

You need correlationId only in one servce in nbWorker. Maybe it is worth to add `correlationId` argument into `saveHistoryAction*` methods?

Reply:
well, I was hesitating about putting it there and I wanted to know other's people opinion on that. It could be a parameter just next to flowPatch, or somehow commandContext could be injected into the operation service class.
Yes, probably it is better to have the correlation ID outside.

Reply:
I removed the variable